stestr/scheduler.py

```diff
@@ -108,7 +108,7 @@ def consume_queue(groups):
             for partition in partitions:
                 temp_part = list(partition)
                 random.shuffle(temp_part)
-                out_parts.append(set(temp_part))
+                out_parts.append(list(temp_part))
```

Comment:
ah, ok. Do you want to rebase this (since github is complaining) and add that to the commit message, then I'll push it through